src/tools/visualStates/generate.cpp

```diff
@@ -637,10 +641,18 @@ void Generate::generateCmake () {
 	this->fs << std::endl;
 	this->fs << "pkg_check_modules(GTKMM REQUIRED gtkmm-3.0)" << std::endl;
 	this->fs << std::endl;
-	this->fs << "SET( INTERFACES_CPP_DIR /opt/jderobot/include )" << std::endl;
+    this->fs << "find_package(roscpp QUIET)" << std::endl;
+    this->fs << "if (roscpp_FOUND)" << std::endl;
+    this->fs << "\tfind_package(catkin REQUIRED COMPONENTS" << std::endl;
+    this->fs << "\t\troscpp" << std::endl << "\t\tstd_msgs" << std::endl << "\t\tcv_bridge" << std::endl;
+    this->fs << "\t\timage_transport" << std::endl << "\t\tnav_msgs" << std::endl << "\t\tgeometry_msgs" << std::endl;
+    this->fs << "\t)" << std::endl << "endif()" << std::endl;
+	this->fs << "SET( INTERFACES_CPP_DIR /usr/local/include )" << std::endl;
 	this->fs << "SET( LIBS_DIR /usr/local/lib )" << std::endl;
-	this->fs << "SET( JDEROBOT_LIBS_DIR /opt/jderobot/lib )" << std::endl;

```

Comment:
Hi @okanasik, 
I've been looking at your changes carefully and you need merge jderobot master branch in yours, because your branch is not updated and reverts many changes of the project.

For example, we have moved JdeRobot installation from /user/* to /opt/ and you revert this

regards, Aitor

Reply:
Thank you for the heads up. I check the visualStates and it seems more than one thing is broken after this change. Therefore, I create a new issue #904 and pull request #905 to solve these problems. With this change we do not write install prefix manually, but get it from cmake.